src/trade_ui.h

```diff
@@ -113,12 +119,11 @@ class trade_ui
         currency_t _cost = 0;
         currency_t _balance = 0;
         std::string const _title;
-        catacurses::window _header_w;
-        ui_adaptor _header_ui;
+        //catacurses::window _header_w;
+        //ui_adaptor _header_ui;
```

Comment:
???

Reply:
Ah yes, these lines were moved from inventory_ui when I moved this class over, I believe. No longer necessary.